src/operator/zip-support.ts

```diff
@@ -56,8 +56,8 @@ export class ZipSubscriber<T, R> extends Subscriber<T> {
     this.active = len;
     for (let i = 0; i < len; i++) {
       let iterator = iterators[i];
-      if (iterator.stillUnsubscribed) {
-        iterator.subscribe(iterator, i);
+      if ((iterator as any).stillUnsubscribed) {
```

Comment:
what about cast type of `iterator` in line 58 once? and is there any feasible type other than `any`?